webnn/resources/utils.js

```diff
@@ -80,32 +80,26 @@ const loadTests = (operationName) => {
 };
 
 /**
- * Get exptected data and data type from given resources with output name.
- * @param {Array} resources - An array of expected resources
+ * Get exptected resource from given resources with output name.
```

Comment:
`exptected`

Reply:
Good catch! Fixed it in new commit, thanks.